mp/src/game/shared/neo/neo_player_shared.h

```diff
@@ -147,9 +147,9 @@ COMPILE_TIME_ASSERT(NEO_ASSAULT_CROUCH_SPEED == NEO_SUPPORT_CROUCH_SPEED);
 // searching until the hulls matched within 1 unit in-game.
 #define NEO_HULL_TOLERANCE_SCALE 1.078125
 
-#define NEO_RECON_MODEL_SCALE ((NEO_RECON_PLAYERMODEL_HEIGHT / HL2DM_DEFAULT_PLAYERMODEL_HEIGHT) * NEO_HULL_TOLERANCE_SCALE)
-#define NEO_ASSAULT_MODEL_SCALE ((NEO_ASSAULT_PLAYERMODEL_HEIGHT / HL2DM_DEFAULT_PLAYERMODEL_HEIGHT) * NEO_HULL_TOLERANCE_SCALE)
-#define NEO_SUPPORT_MODEL_SCALE ((NEO_SUPPORT_PLAYERMODEL_HEIGHT / HL2DM_DEFAULT_PLAYERMODEL_HEIGHT) * NEO_HULL_TOLERANCE_SCALE)
+#define NEO_RECON_MODEL_SCALE 1.f//((NEO_RECON_PLAYERMODEL_HEIGHT / HL2DM_DEFAULT_PLAYERMODEL_HEIGHT) * NEO_HULL_TOLERANCE_SCALE)
+#define NEO_ASSAULT_MODEL_SCALE 1.f//((NEO_ASSAULT_PLAYERMODEL_HEIGHT / HL2DM_DEFAULT_PLAYERMODEL_HEIGHT) * NEO_HULL_TOLERANCE_SCALE)
+#define NEO_SUPPORT_MODEL_SCALE 1.f//((NEO_SUPPORT_PLAYERMODEL_HEIGHT / HL2DM_DEFAULT_PLAYERMODEL_HEIGHT) * NEO_HULL_TOLERANCE_SCALE)
```

Comment:
They seem to all be `1.f` and only used for multiplication, so it seems un-needed now. Any reason to keep them?

Reply:
They are all indeed 1.f, kept them because it looked like a lot of effort went into calculating the previous values is all xd

Reply:
removed unused defs